src/Aspire.Dashboard/Components/Layout/MainLayout.razor.cs

```diff
@@ -87,12 +108,53 @@ public async Task LaunchSettings()
             Height = "auto"
         };
 
-        _ = await DialogService.ShowPanelAsync<SettingsDialog>(parameters).ConfigureAwait(true);
+        await DialogService.ShowPanelAsync<SettingsDialog>(parameters).ConfigureAwait(true);
+    }
+
+    public async Task OnPageKeyDownAsync(KeyboardEventArgsWithPressedKeys args)
+    {
+        if (args is { Key: "?", ShiftKey: true })
+        {
+            await LaunchHelpAsync();
+        }
+        else if (args.Key.ToLower() == "s" && args.ShiftKey)
+        {
+            await LaunchSettingsAsync();
+        }
+        else if (args.CurrentlyHeldKeys.Contains("g"))
+        {
+            if (args.Key.ToLower() == "r")
```

Comment:
We should only call `.ToLower` once in this method, then evaluate it. A switch expression might work here nicely.

Reply:
changed to switch expression, thanks